bin/circleci/build-frontend.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 if [[ " $TESTPILOT_ENABLE_PONTOON_BRANCHES " =~ " $CIRCLE_BRANCH " ]]; then
-    NODE_ENV=production ENABLE_PONTOON=1 npm run static
+    NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_LOCALES=1 npm run static
```

Comment:
So, TL;DR, the vars between local dev & testpilot.dev.mozaws.net are effectively:

* Local - `NODE_ENVIRONMENT=development ENABLE_PONTOON=0 ENABLE_DEV_LOCALES=1`
* Dev  - `NODE_ENVIRONMENT=production ENABLE_PONTOON=1 ENABLE_DEV_LOCALES=1`